content-style-guide.md

```diff
@@ -0,0 +1,87 @@
+# Salt content style guide
+
+Content written for Salt should follow this style guide. This includes component documentation, pattern guides and other website pages.
+
+Use American English and follow general guidance for producing clear content—concise, active voice, second person, present tense.
+
+## Guidance
+
+Entries are listed in alphabetical order.
+
+### Arrow keys
+
+The correct terms for the four arrow buttons on a keyboard are:
+ - <kbd>Up arrow</kbd>
+ - <kbd>Down arrow</kbd>
+ - <kbd>Left arrow</kbd>
+ - <kbd>Right arrow</kbd>
+
+Use \<kbd> tags where appropriate.
+
+### Best practices
+
+Use as a section header where necessary to draw the reader's attention to recommended tips and guidance. Never skip levels when using section headers.
```

Comment:
Should the guidance about section headers be in a more general section than 'Best practices'? And also more specific: `(e.g., h1 followed by h2, h2 followed by h2 or h3, h3 followed by h3 or h4, etc.).` or link to more info like https://www.w3.org/WAI/tutorials/page-structure/headings/